busybee/service.py

```diff
@@ -149,9 +151,23 @@ def register_modules(self):
         print("REGISTERING MODULES")
         print("###############")
         modules = self._mod_descriptors
+
         for module in modules.values():
             module_id = module["id"]
             module_descriptor = module["desc"]
+            if "mod-consortia" in modules and "mod-authtoken" in module_id:
+                env_vars = module_descriptor["launchDescriptor"]["env"]
+                for env_var in env_vars:
+                    if env_var["name"] == "JAVA_OPTIONS":
+                        env_var["value"] += " -Dallow.cross.tenant.requests=true"
+
+            if "mod-consortia" in module_id:
```

Comment:
nitpick: since module_id is a string and not a list, using `in` instead of `==` seems off.

Reply:
As far as I know, in python we can use string.contains method of java as `in` operation. Regarding why I am not using `==`, module name contains snapshot version, so need to use contains operation.

reference first section
https://realpython.com/python-string-contains-substring/

Reply:
I see so it was more of a contains operator. Got it!